nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/TransformXml.java

```diff
@@ -101,16 +101,16 @@
     public static final PropertyDescriptor CACHE_SIZE = new PropertyDescriptor.Builder()
             .name("cache-size")
             .displayName("Cache size")
-            .description("Maximum size of the stylesheet cache.")
+            .description("Maximum number of stylesheets to cache. Zero disables the cache.")
             .required(true)
             .defaultValue("100")
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
             .build();
 
-    public static final PropertyDescriptor CACHE_DURATION = new PropertyDescriptor.Builder()
-            .name("cache-duration")
-            .displayName("Cache duration")
-            .description("How long to keep stylesheets in the cache.")
+    public static final PropertyDescriptor CACHE_TTL_AFTER_LAST_ACCESS = new PropertyDescriptor.Builder()
+            .name("cache-ttl-after-last-access")
+            .displayName("Cache TTL after last access")
+            .description("How long to keep stylesheets in the cache after last access.")
```

Comment:
I like using TTL in the display name in order to make it more compact but not every user may know what TTL means so it'd nice to have the explanation in the description.